chart/openfaas/templates/queueworker-dep.yaml

```diff
@@ -24,7 +24,7 @@ spec:
         {{- if .Values.functionNamespace }}
         env:
         - name: faas_function_suffix
-          value: .{{ .Values.functionNamespace }}
+          value: .{{ .Values.functionNamespace }}.{{ .Values.kubernetesDNSDomain }}
```

Comment:
You missed a dot at the end: `.{{ .Values.functionNamespace }}.{{ .Values.kubernetesDNSDomain }}.`